packages/stores/src/chat/types.ts

```diff
@@ -0,0 +1,208 @@
+export interface NotyphiOrganisations {
+  [key: string]: NotyphiOrganisation;
+}
+export interface NotyphiOrganisation {
```

Comment:
Can we move `Chat`, `Proposals` and `Notyphi` interfaces on different files?
Currently it looks ambiguous in `Chat/types.ts`